Makefile

```diff
@@ -0,0 +1,48 @@
+# Copyright (C) 2018 Red Hat, Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# Setting up paths
+REPOPATH ?= github.com/minishift/minishift-addons
+TEST_DIR ?= $(CURDIR)/testing
+
+GOOS ?= $(shell go env GOOS)
+GOARCH ?= $(shell go env GOARCH)
+ORG := github.com/minishift
+
+# Platfrom dependency
+ifeq ($(GOOS),windows)
+	IS_EXE := .exe
+endif
+
+# Integration tests
+TIMEOUT ?= 3600s
+MINISHIFT_BINARY ?= $(TEST_DIR)/bin/minishift$(IS_EXE)
+
+# Make target definitions
+.PHONY: integration
+integration: ADDON ?= ""
```

Comment:
Need `vendor` as dependent target for `integration`.

Reply:
+1